cwms-data-api/src/main/java/cwms/cda/data/dao/JooqDao.java

```diff
@@ -24,6 +24,8 @@
 
 package cwms.cda.data.dao;
 
+import cwms.cda.api.enums.VersionType;
```

Comment:
VersionType should be unused now. Check for other unused.

Reply:
Removed unused imports